book/src/offline-signing/README.md

```diff
@@ -0,0 +1,78 @@
+# Offline Transaction Signing
+
+Some security models require keeping signing keys, and thus the signing
+process, separated from transaction creation and network broadcast. Examples
+include:
+  * Collecting signatures from geographically disparate signers in a
+multi-signature scheme
+  * Signing transactions using an [airgapped](https://en.wikipedia.org/wiki/Air_gap_(networking))
+signing device
+
+To facilitate these operations, it is necessary to allow transaction
+construction and signing to occur independently.
+
+This document describes using Solana's CLI tools achieve these goals.
+
+## Commands Supporting Offline Signing
+
+At present, the following commands support offline signing:
+  * [`deactivate-stake`](../api-reference/cli.md#solana-deactivate-stake)
+  * [`delegate-stake`](../api-reference/cli.md#solana-delegate-stake)
+  * [`pay`](../api-reference/cli.md#solana-pay)
+
+## Signing Transactions Offline
+
+Signing a transaction offline hinges upon passing two arguments on the command
+line. The first, `--sign-only`, prevents the client from submitting the signed
+transaction to the network. Instead, the pubkey/signature pairs are printed to
+stdout. The second argument, `--blockhash BASE58_HASH`, allows the caller to
+specify the value used to fill the transaction's `recent_blockhash` field. This
```

Comment:
This is fairly text heavy for cli instructions and fairly cli-specific for an explanation paragraph. Maybe split the explanation from the cli args?

Reply:
Yup.  Gonna go with Tyera's suggestion below